IronFoundry.Container.Shared/ProcessRunner.cs

```diff
@@ -116,5 +120,29 @@ public IProcess Run(ProcessRunSpec runSpec)
         public void StopAll(bool kill)
         {
         }
+
+        /// <summary>
+        /// Returns default environment for the process.
+        /// If credentials are specified then the default environment is the default for that user.
+        /// Otherwise the default is to inherit from this process.
+        /// </summary>
+        private Dictionary<string, string> CreateDefaultProcessEnvironment(NetworkCredential credential)
+        {
+            EnvironmentBlock envBlock = new EnvironmentBlock();
+
+            if (credential == null)
+            {
+                envBlock = EnvironmentBlock.Create(Environment.GetEnvironmentVariables());
+            }
+            else
+            {
+                using (var safeUserToken = Utils.LogonAndGetUserPrimaryToken(credential))
+                {
+                    envBlock = EnvironmentBlock.CreateForUser(safeUserToken.DangerousGetHandle());
```

Comment:
Is this really necessary?  If the credentials are for the container user, then are they customized for the container user beyond the system defaults?  Are we relying on getting vars like `HOME` etc?